SETUP/CODE_STYLE.md

```diff
@@ -29,9 +29,24 @@ guide in the wiki.
 
 ## Javascript
 
-We use [eslint](https://eslint.org/) for basic JS code styling and linting.
+We use [prettier](https://prettier.io/) for JS
+code formatting.code styling and linting.
```

Comment:
`formatting.code`? Should that be a comma space instead of a period?

Reply:
oof, yeah. Let me update that tonight.